### DISCUSSION TITLE
Some fields are red and readonly (Mac M2 ARM)

### DISCUSSION THREAD

**Discussion by BenRoe** [comictagger/comictagger#592] [Q&A]
Hi,
i am new and wonder if it's normal that some fields are readonly and red.
<img width="1124" alt="Bildschirmfoto 2024-01-24 um 22 04 49" src="https://github.com/comictagger/comictagger/assets/643834/8c286a60-f3e6-447c-9ae1-d65ca07618f5">

I installed rar via brew with `brew install homebrew/cask/rar`. Original command was `brew install caskroom/cask/rar` but this is deprecated.

**Comment by lordwelch** [comment]
Yes, change the modify style and most of them will go away. The ComicBookInfo format does not support the disabled fields.

**Reply by BenRoe** [reply]
Thanks. That makes sense. I found more info [here](https://code.google.com/archive/p/comictagger/wikis/UserGuide.wiki) under CBL Transforms.
Is there a way to organise the same series in a folder?
